src/Analysis/Ast/Impl/Analyzer/StubMerger.cs

```diff
@@ -95,36 +94,6 @@ internal sealed class StubMerger {
                     continue;
                 }
 
-                var stubPrimaryModule = stubType.DeclaringModule.PrimaryModule;
```

Comment:
Was this removal intended? Seemed like another bugfix.

Reply:
Maybe this code was somewhere else beforehand.

Reply:
Yeah, this was part of a change for caching that was not mergeable. Restored.